### PR TITLE
GPC-347: Add stepfunction for ingest file

### PR DESCRIPTION
**Pull request by oskarwilliams378**

### CODE REVIEW

File: terraform/modules/gro_ingestion_service/ingest_file.tf [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        "Payload.$": "$",
+        "FunctionName": "${aws_lambda_function.split_xml_lambda.arn}:$LATEST"
+      },
+      "Retry": [
+        {
+          "ErrorEquals": [
+            "Lambda.ServiceException",
+            "Lambda.AWSLambdaException",
+            "Lambda.SdkClientException",
+            "Lambda.TooManyRequestsException"
```
Comment (ethanmills) [inline terraform/modules/gro_ingestion_service/ingest_file.tf:27]: Timeout shows up as `"Lambda.Unknown"`

File: terraform/modules/gro_ingestion_service/ingest_file.tf [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+}
+
+resource "aws_s3_bucket_notification" "trigger_stepfunction" {
+  bucket      = module.gro_bucket.id
+  eventbridge = true
+}
+
+resource "aws_cloudwatch_event_rule" "trigger_stepfunction" {
+  name        = "${var.environment}-trigger-gro-ingestion-step-function"
+  description = "Capture each AWS Console Sign In"
```
[the remark this answers is not in the document]
Comment (oskarwilliams378) [reply terraform/modules/gro_ingestion_service/ingest_file.tf:202]: Nope, will fix

File: terraform/modules/gro_ingestion_service/lambdas/src/functions/splitXml.function.ts
[the remark this answers is not in the document]
Comment (oskarwilliams378) [reply terraform/modules/gro_ingestion_service/lambdas/src/functions/splitXml.function.ts:1]: I'm not sure, but I would be happy to attempt for the moment as is, then test it

File: terraform/modules/gro_ingestion_service/ingest_file.tf (hunk 1 above)
Comment (oskarwilliams378) [reply terraform/modules/gro_ingestion_service/ingest_file.tf:27]: Adding

**Review by ethanmills — APPROVED** [review]

**Pull request merged**